about.html

```diff
@@ -8,64 +8,70 @@
-    
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    Let's Work Together
+      <header>
+        <img class="animated tada" src="img/lambda-black.png" alt="lambda logo">
+        <nav>
+          <a href="index.html">Home</a>
+          <a href="about.html">About</a>
+          <a href="products.html">Products</a>
+          <a href="#">Blog</a>
+          <a href="contact.html">Contact</a>
+        </nav>
+      </header>
+
+      <img class="jumbo-image" src="img/jumbo-about.png" alt="cityscape">
+
+      <section class="about-section">
+        <h3>About</h3>
+        <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+      </section>
+
+      <section class="choice-section">
+        <div class="choice">
+          <img src="img/about-plan.png" alt="strategy">
+          <h3>Strategy</h3>
+          <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+          <button>Learn More</button>
+        </div>
+        <div class="choice">
+          <img src="img/about-working.png" alt="strategy">
+          <h3>How We Work</h3>
+          <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+          <button>Learn More</button>
+        </div>
+        <div class="choice">
+          <img src="img/about-office.png" alt="strategy">
+          <h3>Places We Work</h3>
+          <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+          <button>Learn More</button>
+        </div>
+        <div class="choice">
+          <img src="img/about-meeting.png" alt="strategy">
+          <h3>Collaboration</h3>
+          <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+          <button>Learn More</button>
+        </div>
+      </section>
+
+      <section class="work-together">
+        <h3>Let's Work Together</h3>
+        <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+      </section>
 
```

Comment:
I like the naming of your classes! It makes it easier for later programmers to understand your code/styling :)